gateway-ha/pom.xml

```diff
@@ -97,6 +97,22 @@
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>io.airlift</groupId>
+            <artifactId>http-client</artifactId>
```

Comment:
are we potentially removing OkHttp to use this in the future? Or we will keep both in the repo?